src/features/completionItemProvider.ts

```diff
@@ -79,14 +79,27 @@ export default class OmniSharpCompletionItemProvider extends AbstractSupport imp
 }
 
 const _kinds: { [kind: string]: CompletionItemKind; } = Object.create(null);
-_kinds['Variable'] = CompletionItemKind.Variable;
-_kinds['Struct'] = CompletionItemKind.Interface;
-_kinds['Interface'] = CompletionItemKind.Interface;
-_kinds['Enum'] = CompletionItemKind.Enum;
-_kinds['EnumMember'] = CompletionItemKind.Property;
-_kinds['Property'] = CompletionItemKind.Property;
+
+// types
 _kinds['Class'] = CompletionItemKind.Class;
+_kinds['Delegate'] = CompletionItemKind.Class; // need a better option for this.
+_kinds['Enum'] = CompletionItemKind.Enum;
+_kinds['Interface'] = CompletionItemKind.Interface;
+_kinds['Struct'] = CompletionItemKind.Class; // need a better option for this.
+
+// variables
+_kinds['Local'] = CompletionItemKind.Variable;
+_kinds['Parameter'] = CompletionItemKind.Variable;
+_kinds['RangeVariable'] = CompletionItemKind.Variable;
+
+// members
+_kinds['EnumMember'] = CompletionItemKind.Property; // need a better option for this.
+_kinds['Event'] = CompletionItemKind.Field; // need a better option for this.
```

Comment:
I believe there is a long standing issue to add an event icon to that completion SVG in VS Code repo. Quite a shame it's still not there